File: Day-11/Pattern13.js
function PatternThirteen(n) {
  let i = 1;
  let ch = 65;

  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= n) {
      str += String.fromCharCode(ch);
      j++;
      ch++;
    }
    i++;
    console.log(str);
  }
}

PatternThirteen(3);

File: Day-11/Pattern13.py
n = int(input())

def PatternThirteen(n):
    i = 1
    ch = 65
    while(i <= n):
        j = 1
        while(j <= n):
            print(chr(ch), end='')
            ch += 1
            j += 1
        print(end='\n')
        i += 1

PatternThirteen(n)